src/data/rat/field.lean

```diff
@@ -0,0 +1,131 @@
+/-
+Copyright (c) 2019 Johannes Hölzl. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johannes Hölzl, Mario Carneiro, Yaël Dillies
+-/
+import algebra.order.nonneg.field
+import tactic.nth_rewrite
+
+/-!
+# Field Structure on the (nonnegative) Rational Numbers
+
+> THIS FILE IS SYNCHRONIZED WITH MATHLIB4.
+> Any changes to this file require a corresponding PR to mathlib4.
```

Comment:
Why is this in a new file?

Reply:
It was moved from `data.rat.basic`. My reasoning is that this is mostly about the field instance and nothing else, but I can revert if you want. I'm not particularly attached to the rename (it used to make more sense because I also had a `data.rat.nnrat.field` file).

Reply:
And my answer was "because the file just got moved" 😄